app/components/CoachFilterTag.tsx

```diff
@@ -18,8 +19,11 @@ export default function CoachFilterTag({
   children,
   type,
 }: CoachFilterTagProps) {
+  const isDeepCrisisTag = (): boolean | undefined => {
+    return children?.toString().includes("1.");
```

Comment:
One thing that came up in my mind this afternoon: Let's hope no other tag will start with `1.` in the future :D This is absolutely valid for now. In general, I'd rather abstract this away e.g. into a boolean prop like `isHighlighted` that is checked higher up in the component tree against the `id`/`value` of the tag, for example.

Reply:
Great point, thanks! I'm updating it. I will use the id, as the value will change by language. 

I'm also adding an extra check to sort the tags array and ensure this one is the first one, otherwise it will break the array if we ever remove the `1.`, will ask you to re-review it later because I'm not sure if where I have added it is the best spot 🤔